README.md

```diff
@@ -10,15 +10,51 @@ With the public release of `$vectorSearch`, we have needed to integrate into the
 ([LangChain](https://github.com/langchain-ai/langchainjs), [LlamaIndex](https://github.com/run-llama/llama_index), [Semantic Kernel](https://github.com/microsoft/semantic-kernel)... etc) This repository runs continuous testing against each of these repos.
 
 ## How to add a test
+> **NOTE** All tests run against this repo are now required to work against a local atlas deployment. See details below to ensure proper set-up
 
 ### Test Layout
 
 Each AI/ML pipeline is sorted by the composite of the name of the library, and the driver language the library is implemented in. This comes out in the format `{pipeline}-{language}` --> `semantic-kernel-python`. All tests should be scoped within the bounds of these subdirectories.
 
-Within each subdirectory you should expect to have a `run.sh` file. This run script will handle any additional installation, or scaffolding steps to get the test suite running. Note, it does not need to include steps to scaffold the atlas database. Additionally, you can add useful environment files such as a `.env` file or atlas vector search index configurations.
+Within each subdirectory you should expect to have:
+- `run.sh`  --  This run script will handle any additional library installation, or steps to get the test suite running. Note, it does not need to include steps to scaffold the atlas database.
+- `database/` -- Optional directory where the `.evergreen/scaffold_atlas.py` will read and use to population a mongo database with collection information. Only provide this if your tests assume a pre-populated database
+- `database/{collection}.json` -- JSON file with the mongo documents that will be upload to `$DATABASE.{collection}`
+- `indexConfig.json` -- File with Atlas Search Index configuration.
+- Additionally, you can add useful environment files such as a `.env` file.
 
 Each subdirectory holds the space for one repository, and within each subdirectory is where evergreen will clone the specified third-party library.
```

Comment:
I don't quite understand what this line is saying, can you explain?

Reply:
How about
"""
Each subdirectory will only clone one AI/ML Integration repo. Each subdirectory is scoped to run only one AI/ML Integration's suite of tests for one language within that cloned repository. For example, if an AI/ML Integration has both a Python and C# implementation of Atlas Vector Search, two subdirectories need to be made: one for Python, titled `{repo}-python`, and one for C#, titled `{repo}-csharp`. See `semantic-kernel-*` subdirectories in the layout example below.
"""

Reply:
Here, I'd want to err on the side of redundancy. The new revised speaks more explicitly and directly towards that use-case.